docs/Catalogs.adoc

```diff
@@ -0,0 +1,279 @@
+= Using Operator Catalog Mirrors
```

Comment:
I think you accidentally included this file in the PR here when it shoudl be in #2222

Reply:
Removed